tasks/browserify.js

```diff
@@ -29,6 +30,11 @@ module.exports = function (gulp, plugins, options) {
       ignore: ['jquery']
     });
 
+    // The ignore is for the jQuery stream below
+    bundler.transform(babelify.configure({
+      ignore: /_stream_0\.js/
```

Comment:
We should probably either add a `blacklist` or `whitelist` option here to ensure we only use transformations that don't blow up in IE8.